utils/init-google-map.ts

```
@@ -78,15 +78,19 @@ export const initGoogleMap = async (probe: Probe) => {
 		minZoom: MAP_MIN_ZOOM,
 		maxZoom: MAP_MAX_ZOOM,
 		gestureHandling: 'cooperative',
 	});
 
-	createMapMarker(probe);
+	createMapMarker(probe, showPulse, showIW);
 
 	map.addListener('zoom_changed', () => {
-		infoWindow && infoWindow.close();
+		if (showIW) {
+			infoWindow && infoWindow.close();
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)